charts/trino/templates/configmap-worker.yaml

```diff
@@ -83,10 +83,24 @@ data:
   {{- end }}
 {{ end }}
 
+
+{{- if .Values.accessControl }}
+  {{ include "trino.accessControlProperties" . }}
+{{- end }}
+
 {{- range $fileName, $fileContent := .Values.worker.additionalConfigFiles }}
   {{ $fileName }}: |
     {{- $fileContent | nindent 4 }}
 {{- end }}
+
+---
+
+{{- if .Values.accessControl }}
+  {{- if eq .Values.accessControl.type "configmap" }}
+  {{ include "trino.accessControlConfigMap" . }}
```

Comment:
You're redefining the same configmap here, we only need one. Keep this in a separate file.

Reply:
There's a failure in the workflow run caused by this: https://github.com/trinodb/charts/actions/runs/10656311173/job/29534820889?pr=213